File: 프로그래머스/lv2.멀쩡한 사각형.js
// Level2: 멀쩡한 사각형
function solution(w, h) {
    const gcd = function(a,b){
        if(b === 0) return a;
        if(a % b === 0) return b;
        else return gcd(b,a%b);
    }
    const GCD = gcd(w,h);
    return w*h - (w+h-GCD);
}

File: 프로그래머스/lv2.멀쩡한 사각형.py
# Level2: 멀쩡한 사각형
def solution(w,h):
    answer = 1
    def GCD(a,b):
        if b == 0: return a
        if a % b == 0: return b
        else: return GCD(b,a % b)
    gcd = GCD(w,h)
    return w*h - (w+h-gcd)